src/cmake/android_openssl.cmake

```diff
@@ -0,0 +1,61 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+
```

Comment:
nit: Too many whitespaces throughout the whole file. If intentional please ignore me.